frontend/src/components/notifications/NotificationDetails.vue

```diff
@@ -1,13 +1,14 @@
+<!-- eslint-disable vue/no-v-html -->
 <template>
   <v-card v-if="notification" variant="flat">
     <v-card-title class="notification-details-header">
       <v-row>
-        <v-col sm="12" lg="9" xl="10" class="notification-from-text pa-0">
+        <v-col cols="7" md="8" lg="9" class="notification-from-text pa-0">
           <strong>From:</strong>
           &nbsp;Operating Funding Model Program
         </v-col>
-        <v-col sm="12" lg="3" xl="2" class="pa-0 d-flex justify-end mt-1">
-          <AppButton v-if="notification?.isRead" variant="text" @click="$emit('toggleMarkUnreadButtonInNotificationDetails')">
+        <v-col cols="5" md="4" lg="3" class="pa-0 d-flex justify-end mt-1">
```

Comment:
it would be great if we can keep the distance between text and icon the same for all buttons (in Messages and Notifications)